CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 # How to Contribute
 
 We'd love to accept your patches and contributions to this project.
-DSP can be subtle to get completely right, so we particularly appreciate the
```

Comment:
DSP is actually intended here.

Reply:
Reverted to DSP in https://github.com/magenta/ddsp/pull/1/commits/13c29ccbb56a5431eaa424bc79bf1bfa77898a76. Thank you!